plugins/osdn/ovs/bin/openshift-sdn-ovs

```diff
@@ -83,6 +94,17 @@ del_ovs_flows() {
     ovs-ofctl -O OpenFlow13 del-flows br0 "ip,nw_src=${ipaddr}"
     ovs-ofctl -O OpenFlow13 del-flows br0 "arp,nw_dst=${ipaddr}"
     ovs-ofctl -O OpenFlow13 del-flows br0 "arp,nw_src=${ipaddr}"
+
+    qos=$(ovs-vsctl list port ${veth_host} >&/dev/null | grep qos | cut -d ":" -f 2)
+    if [ -n "$qos" ]; then
+        queue=$(ovs-vsctl list qos ${qos} | grep queues | cut -d ":" -f 2 | cut -d "=" -f 2 | cut -d "}" -f 1)
```

Comment:
maybe other people don't like regexes, but I feel like using sed here would be simpler:

    queue=$(ovs-vsctl get QoS ${qos} queues | sed -e 's/.*\([0-9]*\)=.*/\1/')

(not tested since I can't figure out how to make queues end up non-empty)